Tools/PowershellModule/doc/new-private-repository-azure.md

```diff
@@ -1,238 +1,226 @@
-| No       | SubscriptionName           | The name of the Azure Subscription that will be used to pay for the Azure resources.                                                       |
-| No       | Region                     | The Azure location where the Azure resources will be created. (Default: westus)                                                            |
-| No       | ParameterOutput            | The folder location that contains new items will be created in.                                                                            |
-| No       | RestSourcePath             | Path to the compiled REST API Zip file. (Default: .\Library\RestAPI\WinGet.RestSource.Functions.zip)                                                         |
-| No       | ImplementationPerformance  | specifies the performance of the resources to be created for the Windows Package Manager REST source. ["Demo", "Basic", "Enhanced"]        |
-| No       | ShowConnectionInstructions | If specified, the instructions for connecting to the Windows Package Manager REST source. (Default: False)                                 |
-
-> [!Note]
-> The PowerShell Module must be re-imported each time the PowerShell window is closed.
+| Required | Parameter                  | Description                                                                                                                         |
+|----------|----------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
+| Yes      | Name                       | A string of letters which will be prefixed to your newly created Azure resources.                                                   |
+| Yes      | ResourceGroup              | The Resource Group that will be used to contain the Azure resources.                                                                |
+| No       | SubscriptionName           | The name of the Azure Subscription that will be used to pay for the Azure resources.                                                |
+| No       | Region                     | The Azure location where the Azure resources will be created. (Default: westus)                                                     |
+| No       | ParameterOutput            | The folder location where the ARM parameter files will be created.                                                                  |
+| No       | RestSourcePath             | Path to the compiled Rest API Zip file. (Default: .\RestAPI\CompiledFunctions.ps1)                                                  |
+| No       | ImplementationPerformance  | specifies the performance of the resources to be created for the Windows Package Manager rest source. ["Demo", "Basic", "Enhanced"] |
+| No       | ShowConnectionInstructions | If specified, the instructions for connecting to the Windows Package Manager rest source. (Default: False)                          |
```

Comment:
Description is fragmented.  Needs to clarify that this means the command will show/output the instructions.